collect_app/src/test/java/org/odk/collect/android/utilities/MainActivityTest.java

```diff
@@ -0,0 +1,165 @@
+package org.odk.collect.android.utilities;
```

Comment:
I think we should match the names and package structure of the actual classes that are being tested. In this case, it would be nice to have this be `MainMenuActivityTest` in the `activity` package. Why is it in `utilities`? I think that might have accidentally have been copied from `TextUtilsText`.